process/wg-lifecycle-documents/Global_Cyber_Policy_WG_sandbox_stage.md

```diff
@@ -0,0 +1,40 @@
+## Application for creating a new Working Group at Sandbox stage
+
+Global Cyber Policy Working Group
+
+### List of interested individuals
+The WG must have a minimum of 3 interested contributors from two different organizations.
+  * Arnaud Le Hors, IBM, [lehors](https://github.com/lehors)
+  * CRob, OpenSSF-LF, [SecurityCRob](https://github.com/SecurityCRob)
+  * Daniel Appelquist, Samsung, [Torgo](https://github.com/Torgo)
+  * Georg Kunz, Ericsson, [gkunz](https://github.com/gkunz)
+  * Christian "fukami" Horchert, OpenSSF-LF, [ ]( )
+  * Mirko Boehm, LF-Europe, [ ]( )
+
+
+### Mission of the Working Group
+The WG must be aligned with the OpenSSF mission and address an unfulfilled need. It is preferred that topics falling with the scope of existing OpenSSF WGs are addressed within the existing wG rather than seek a new WG.
+  * To provide a forum for our members and the broader community to collaborate on Global Cybersecurity-related legislation, frameworks, and standards, this group will begin with a focus on the EU's CRA legislation, but in the future can expand into other related topics.
+  * The First three SIGs that will be created will be:
+  1.) CRA-Awareness - focused on educational and training materials, research and events that help educate Manufacturers, Stewards, and open source maintainers on obligations, artifacts, and processes they must have in place to comply with the CRA.
+  2.) CRA-Community Standards - focused on converting applicable community specifications into internationally-recognized standards.
+  3.) CRA-Format, Tooling & Process - focused on data formats, tooling and processes that must be put into place to help Linux Foundation projects be compliant with the CRA.
```

Comment:
It sounds like the scope for this SIG is broader than OpenSSF? I'm assuming there will also be a lot of cross-collaboration with other OSSF TIs that focus on tooling, data formats, etc..